var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -17,6 +17,7 @@ class Hiop(CMakePackage, CudaPackage):
     git = "https://github.com/LLNL/hiop.git"
 
     # Most recent tagged snapshot is the preferred version when profiling.
+    version('0.4.1', tag='v0.4.1')
```

Comment:
Confirmed tag exists.  

FYI. Hashes are preferred over tags.

Reply:
Thanks! Could you point me to some documentation about this? Why do you prefer hashes?